packages/model/db.go

```diff
@@ -28,6 +28,27 @@ var (
 
 	// ErrDBConn database connection error
 	ErrDBConn = errors.New("Database connection error")
+
+	FirstEcosystemTables = map[string]bool{
```

Comment:
why not use just map[string]struct{}, and check key existance in it?

Reply:
For _keys and _memebers it equals 'false' because they don't have unique id key. It equals 'true' for all other tables. These boolean values are used in other places.